### PR TITLE
Added Swahili Translations

### PR DESCRIPTION
**Pull request by wambugucoder**
> Make sure you go through the [Contributing guide](https://docs.legesher.io/the-official-things/contributing-guidelines).

> Please delete options that are not relevant.

## Title

This pull request (PR) is made in reference to:

## Description

> What does this PR do?

> Recommendations for how to test this?

## Motivation and Context

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Testing

- [ ] My submission passes the tests?
  - If you did additional testing, please describe in detail how you tested your changes
- [ ] I added new tests for core changes, as applicable?

### CODE REVIEW

**Review by madiedgar — APPROVED** [review]
Looks amazing! Thank you @wambugucoder 👍

**Comment by wambugucoder** [comment]
I'm always happy to help @madiedgar

File: locale/sw.yml [hunk 1]
```diff
@@ -0,0 +1,1813 @@
...
+  del: "duta"
+  elif: "vinginevyo"
+  else: "ikiwa-si-ivyo"
+  except: "isipokuwa"
+  finally: "mwishowe"
+  for: "kwa"
+  from: "kutoka"
+  global:  "kimataifa"
+  if: "kama"
+  import: "agiza"
```
Comment (Sanqqara) [inline locale/sw.yml:1272]: import: "leta"

File: locale/sw.yml [hunk 2]
```diff
@@ -0,0 +1,1813 @@
...
+  in: "ndani"
+  is: "ni"
+  lambda: "lambda"
+  nonlocal: "isiyo-ya-kawaida"
```
Comment (Sanqqara) [inline locale/sw.yml:1276]: nonlocal: We should find a word that fits the actual semantic meaning of nonlocal.

File: locale/sw.yml [hunk 3]
```diff
@@ -0,0 +1,1813 @@
...
+  not: "si"
+  or: "au"
+  pass: "pita"
+  raise: "inua"
```
Comment (Sanqqara) [inline locale/sw.yml:1280]: raise: "leta"

File: locale/sw.yml [hunk 4]
```diff
@@ -0,0 +1,1813 @@
...
+  return: "rudisha"
```
Comment (Sanqqara) [inline locale/sw.yml:1281]: return: "rejesha"

File: locale/sw.yml [hunk 5]
```diff
@@ -0,0 +1,1813 @@
...
+  try: "jaribu"
+  while: "wakati"
+  with: "na"
```
Comment (Sanqqara) [inline locale/sw.yml:1284]: Conflict of translation with "and".
I feel we should change and to "pamoja".

File: locale/sw.yml [hunk 6]
```diff
@@ -0,0 +1,1813 @@
...
+  yield: "mavuno"
```
Comment (Sanqqara) [inline locale/sw.yml:1285]: yield: We should find a word that fits the actual semantic meaning of yield.

File: locale/sw.yml [hunk 7]
```diff
@@ -0,0 +1,1813 @@
...
+  all: "yote"
+  any: "yoyote"
+  ascii: "ascii"
+  bin: "bin"
+  bool: "bool"
+  breakpoint: "mahali-pa-kuvunja"
+  bytearray: "baiti-safu"
+  bytes: "baiti"
+  callable: "inayoitwa"
+  chr: "tabia"
```
Comment (Sanqqara) [inline locale/sw.yml:1299]: "tabia" may be losing the literal meaning of the word don't you think?

File: locale/sw.yml [hunk 8]
```diff
@@ -0,0 +1,1813 @@
...
+
+# << Built-In Functions >>
+
+  abs: "abs"
+  all: "yote"
+  any: "yoyote"
+  ascii: "ascii"
+  bin: "bin"
+  bool: "bool"
+  breakpoint: "mahali-pa-kuvunja"
```
Comment (Sanqqara) [inline locale/sw.yml:1295]: I suggest we think of reducing the use of dashed words.

Breakpoint could just be "breakpointi" 😂

File: locale/sw.yml [hunk 9]
```diff
@@ -0,0 +1,1813 @@
...
+  bytearray: "baiti-safu"
+  bytes: "baiti"
+  callable: "inayoitwa"
+  chr: "tabia"
+  classmethod: "njia-ya-darase"
```
Comment (Sanqqara) [inline locale/sw.yml:1300]: classmethod: "namnajumuiya"

File: locale/sw.yml [hunk 10]
```diff
@@ -0,0 +1,1813 @@
...
+  compile: "kusanya"
+  complex: "tata"
+  delattr: "futa-sifa"
```
Comment (Sanqqara) [inline locale/sw.yml:1303]: Dash use?

File: locale/sw.yml [hunk 11]
```diff
@@ -0,0 +1,1813 @@
...
+  property: "mali"
+  range: "masafa"
+  repr: "repr"
+  reversed: "geuza-nyuma"
+  round: "raundi"
+  set: "seti"
+  setattr: "weka-sifa"
+  slice: "kipande"
+  sorted: "pangwa"
+  staticmethod: "njia-tuli"
```
Comment (Sanqqara) [inline locale/sw.yml:1349]: Dashes?

File: locale/sw.yml [hunk 12]
```diff
@@ -0,0 +1,1813 @@
...
+  ord: "utaratibu"
+  pow: "nguvu"
+  print: "chapisha"
+  property: "mali"
+  range: "masafa"
+  repr: "repr"
+  reversed: "geuza-nyuma"
+  round: "raundi"
+  set: "seti"
+  setattr: "weka-sifa"
```
Comment (Sanqqara) [inline locale/sw.yml:1346]: Dashes?

File: locale/sw.yml [hunk 13]
```diff
@@ -0,0 +1,1813 @@
...
+  object: "kitu"
+  oct: "oct"
+  open: "fungua"
+  ord: "utaratibu"
+  pow: "nguvu"
+  print: "chapisha"
+  property: "mali"
+  range: "masafa"
+  repr: "repr"
+  reversed: "geuza-nyuma"
```
Comment (Sanqqara) [inline locale/sw.yml:1343]: Dashes?

File: locale/sw.yml [hunk 14]
```diff
@@ -0,0 +1,1813 @@
...
+  isinstance: "mfano"
+  issubclass: "darasa-ndogo"
+  iter: "mara-kwa-mara"
+  len: "urefu"
+  list: "orodha"
+  locals: "wenyeji"
+  map: "ramani"
+  max: "upeo"
+  memoryview: "mtazamo-wa-kumbukumbu"
+  min: "kiwango-cha-chini"
```
Comment (Sanqqara) [inline locale/sw.yml:1332]: min: "ndogo"

File: locale/sw.yml [hunk 15]
```diff
@@ -0,0 +1,1813 @@
...
+  next: "ijayo"
+  object: "kitu"
+  oct: "oct"
+  open: "fungua"
+  ord: "utaratibu"
+  pow: "nguvu"
+  print: "chapisha"
+  property: "mali"
+  range: "masafa"
```
Comment (Sanqqara) [inline locale/sw.yml:1341]: range: "kati"

File: locale/sw.yml [hunk 16]
```diff
@@ -0,0 +1,1813 @@
...
+  hex: "hex"
+  id: "kitambulisho"
+  input: "pembejeo"
+  int: "nambari"
+  isinstance: "mfano"
+  issubclass: "darasa-ndogo"
+  iter: "mara-kwa-mara"
+  len: "urefu"
+  list: "orodha"
+  locals: "wenyeji"
```
Comment (Sanqqara) [inline locale/sw.yml:1328]: Up for discussion...

File: locale/sw.yml [hunk 17]
```diff
@@ -0,0 +1,1813 @@
...
+  exec: "tekeleza"
+  filter: "chuja"
+  float: "elea"
+  format: "muundo"
+  frozenset: "waliohifadhiwa"
+  getattr: "pata-sifa"
+  globals: "ulimwengu"
+  hasattr: "ina-sifa"
+  hash: "hash"
+  help: "saidia"
```
Comment (Sanqqara) [inline locale/sw.yml:1318]: help: "usaidizi"

File: locale/sw.yml [hunk 18]
```diff
@@ -0,0 +1,1813 @@
...
+  complex: "tata"
+  delattr: "futa-sifa"
+  dict: "kamusi"
+  dir: "saraka"
+  divmod: "divmod"
+  enumerate: "kuhesabu"
+  eval: "tathmini"
+  exec: "tekeleza"
+  filter: "chuja"
+  float: "elea"
```
Comment (Sanqqara) [inline locale/sw.yml:1311]: Could the meaning of this word be getting lost in translation?

**Review by Sanqqara — COMMENTED** [review]
Hey @wambugucoder check out these suggestions. I'm open to discuss them.

**Comment by madiedgar** [comment]
Hey @Sanqqara, these are great suggestions. Could you create a pull request with all your reviews? That was future reviewers can be part of the conversation as well? 👍 

[Take a look at the Python translation from another language's branch for an example](https://github.com/legesher/legesher-translations/blob/portuguese-translation/locale/pt.yml). Let me know if you have any questions!

**Comment by Sanqqara** [comment]
@madiedgar sure I'll be on it.

File: locale/sw.yml (hunk 2 above)
Comment (collinskoech11) [reply locale/sw.yml:1276]: nonlocal : isiyo-ya-ndani

**Pull request merged**